nuxeo-services/login/nuxeo-platform-login-keycloak/src/main/java/org/nuxeo/ecm/platform/ui/web/keycloak/KeycloakAuthenticationPlugin.java

```diff
@@ -173,7 +173,10 @@ private KeycloakUserInfo getKeycloakUserInfo(AccessToken token) {
      */
     private Set<String> getRoles(AccessToken token, String keycloakNuxeoApp) {
         Set<String> allRoles = new HashSet<>();
-        Set<String> roles = token.getRealmAccess().getRoles();
+        Set<String> roles = null;
+        if (token.getRealmAccess() != null) {
+            roles = token.getRealmAccess().getRoles();
+        }
         if (roles != null) {
             allRoles.addAll(roles);
         }
```

Comment:
We can probably do the following, can you check?
```suggestion
        if (token.getRealmAccess() != null) {
            allRoles.addAll(roles);
        }
```

Reply:
Yes it also works, I'll amend my commit